src/components/sections/pricing/comparison/components/heading-plan/heading-plan.styles.ts

```diff
@@ -15,8 +15,8 @@ export const StyledHeadingPlanWrapper = styled.div`
       width: 100%;
     }
 
-    & button,
-    & a {
+    & > * > button,
+    & > * > a {
```

Comment:
mmm not sure about this approach. If we want to target the button/a in the dropdown button, we can use data attributes to be more accurate. Something like:

``` ts
    & button[data-drowpdown="true"],
    & a[data-drowpdown="true"] {
    ...
    }
```

wdyt?